README.md

```diff
@@ -1 +1,72 @@
 ### Google Homepage
+
+## Installation & Usage
+
+### Installation
+
+-   Clone or download the repo.
+-   Open terminal and navigate to `google-homepage` folder.
+-   Run `npm install` to install dependencies.
+
+### Usage
+
+-   Run `npm run dev` to launch server using nodemon
+
+## Changelog
```

Comment:
keeping the tracker of changes you have made would be helpful to understand your though process and perhaps troubleshooting the code.